Kentico.Kontent.Delivery/DeliveryClient.cs

```diff
@@ -93,7 +93,13 @@ public async Task<JObject> GetItemJsonAsync(string codename, params string[] par
         /// <returns>The <see cref="JObject"/> instance that represents the content items. If no query parameters are specified, all content items are returned.</returns>
         public async Task<JObject> GetItemsJsonAsync(params string[] parameters)
         {
-            var endpointUrl = UrlBuilder.GetItemsUrl(parameters);
+            var parameterList = parameters?.ToList() ?? new List<string>();
+            if (DeliveryOptions.IncludeTotalCount && !parameterList.Contains(new IncludeTotalCountParameter().GetQueryStringParameter()))
```

Comment:
Perhaps this piece of logic can be refactored into a method? It is used multiple times and if someone decides to add a new parameter with an optional global value, it will probably only get worse.